yapapi/package/vm.py

```diff
@@ -62,8 +64,16 @@ async def repo(
     - *min_storage_gib* minimal disk storage to execute tasks.
 
     """
+
     return _VmPackage(
-        repo_url=_DEFAULT_REPO_URL,
+        repo_url=resolve_repo_srv(_DEFAULT_REPO_SRV),
         image_hash=image_hash,
         constraints=_VmConstrains(min_mem_gib, min_storage_gib),
     )
+
+
+def resolve_repo_srv(repo_srv) -> str:
+    srv: SRVRecord = SRVResolver.resolve_random(repo_srv)
```

Comment:
I've checked that `SRVResolver.resolve_random()` raises exceptions if `repo_srv` is invalid or network is down. Maybe we should wrap those exceptions in our `PackageException`.